src/walkthrough/README.md

```diff
@@ -0,0 +1,13 @@
+Developer Walkthroughs
```

Comment:
We are trying to follow the pattern to put documents into the docs folder at the top-level. The READMEs about folder structure and what each component does are fine to be in the folders like they are currently. But we should move anything instructional items like this to docs folder.

Reply:
I thought about putting these in "docs" but it felt like there might be other more generic docs that would go there (setting up your environment, etc).

Can we just approve this one and merge it?  Then I'll make this change in the next branch that is already off of this one.... and has 2 walkthrus still waiting on review, plus one more in progress. Waiting so long for PRs when things build off each other causes a lot of rework.

Reply:
I think doing it in later iteration is ok. But I would also suggest not to create PRs from existing branches to avoid rework.

Reply:
Yes, unrelated branches would be better. But when the work builds on top of previous work, that is very hard to do. At a minimum, in the case of the next walkthoughs, I would need to update the "menu" page listing all of the available walkthrough docs. If this PR is not already in master, then that page is not available for me to add on to for the next docs. Which then causes different merge issues if I add the file in both branches.  Yes, some advanced planning might help with this, but will likely not alleviate it completely. Faster turn around on reviews would also help.